currency-conversion/src/main/java/lk/ac/ucsc/currencyconversion/CurrencyConversionController.java

```diff
@@ -0,0 +1,47 @@
+package lk.ac.ucsc.currencyconversion;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class CurrencyConversionController {
+
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
@Binurim loggers should be static and final

Reply:
[Done]
(https://stackoverflow.com/questions/6653520/why-do-we-declare-loggers-static-final)
(https://stackoverflow.com/questions/33779127/loggerfactory-getloggerclassname-class-vs-loggerfactory-getloggerthis-getclas/33779432)